### PR TITLE
fix(web): image upload

### PR DESCRIPTION
**Pull request by samacamelia**
# Description

Please provide a detailed description of what was done in this PR.
Precise the issue that you are resolving.

# Fixes

Specify which issue this PR targets, like so:

- fixes #0

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added @vrn-sh/members to the reviewers, or specific members of the team
- [ ] I have added the needed labels
- [ ] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

File: web/vite.config.ts.timestamp-1695071566216-e680af4895c7b.mjs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// vite.config.ts
+import { defineConfig, loadEnv } from "file:///mnt/d/School/EIP/lastest/erp/web/node_modules/vite/dist/node/index.js";
```
Comment (djnnvx) [inline web/vite.config.ts.timestamp-1695071566216-e680af4895c7b.mjs]: you need to remove this file i think

**Review by djnnvx — COMMENTED** [review]
what does this PR fix ?

File: web/vite.config.ts.timestamp-1695071566216-e680af4895c7b.mjs (hunk 1 above)
Comment (samacamelia) [reply web/vite.config.ts.timestamp-1695071566216-e680af4895c7b.mjs]: erp 185 le telechargement des images au niveau des vulnerabilité

Comment (samacamelia) [reply web/vite.config.ts.timestamp-1695071566216-e680af4895c7b.mjs]: je l'ai supprimé cest bon

File: web/src/pages/Vulnerability/EditVuln.tsx [hunk 2]
```diff
@@ -44,26 +46,60 @@ export default function AddVulnerability() {
...
+    
+            // Comptez combien d'images sont déjà ajoutées
+            const existingImageCount = newImages.filter((image) => !!image).length;
+    
+            // Compteur pour suivre le nombre d'images ajoutées
+            let addedImageCount = 0;
+    
+            for (let i = 0; i < fileList.length; i += 1) {
+                if (addedImageCount + existingImageCount >= MAX_IMAGES) {
+                    console.log('max atteint');
```
Comment (Ananane) [inline web/src/pages/Vulnerability/EditVuln.tsx]: /!\ comments

**Review by Ananane — COMMENTED** [review]
Looks good to me but would be perfect with the help of your linter!

**Comment by djnnvx** [comment]
image

linter will not work

**Pull request merged**